junifer/markers/crossatlas_functional_connectivity.py

```diff
@@ -0,0 +1,164 @@
+        (default "pearson").
+    name : str, optional
+        The name of the marker. If None, will use the class name
+        (default None).
+    """
+
+    def __init__(
+        self,
+        atlas_one: str,
+        atlas_two: str,
+        aggregation_method: str = "mean",
+        correlation_method: str = "pearson",
+        name: str = None,
+    ) -> None:
+        """Initialize the class."""
+        self.atlas_one = atlas_one
+        self.atlas_two = atlas_two
+        self.aggregation_method = aggregation_method
+        self.correlation_method = correlation_method
+        super().__init__(on=["BOLD"], name=name)
+
+    def get_valid_inputs(self) -> List[str]:
+        """Get valid data types for input.
+
+        Returns
+        -------
+        list of str
+            The list of data types that can be used as input for this marker
+
+        """
+        return ["BOLD"]
+
+    def get_output_kind(self, input: List[str]) -> List[str]:
+        """Get output kind.
+
+        Parameters
+        ----------
+        input : list of str
+            The input to the marker. The list must contain the
+            available Junifer Data dictionary keys.
+
+        Returns
+        -------
+        list of str
+            The updated list of output kinds, as storage possibilities.
+
+        """
+        return ["matrix"]
+
+    def store(
+        self,
+        kind,
+        out: Dict[str, Any],
+        storage: "BaseFeatureStorage",
+    ) -> None:
+        """Store.
+
+        Parameters
+        ----------
+        kind : {"BOLD"}
+            The data kind to store.
+        out : dict
+            The computed result as a dictionary to store.
+        storage : storage-like
+            The storage class, for example, SQLiteFeatureStorage.
+
+        """
+        logger.debug(f"Storing BOLD-based marker in {storage}")
+        storage.store(kind="matrix", **out)
+
+    def compute(
+        self,
+        input: Dict[str, Any],
+        extra_input: Optional[Dict] = None,
+    ) -> Dict:
+        """Compute.
+
+        Take a timeseries, parcellate them with two different parcellation
+        schemes, and get parcel-wise correlations between the two different
+        parcellated time series. Shape of output matrix corresponds to number
+        of ROIs in (atlas_two, atlas_one).
+
+        Parameters
+        ----------
+        input : dict
+            The BOLD data as a dictionary.
+        extra_input : dict, optional
+            The other fields in the pipeline data object (default None).
+
+        Returns
+        -------
+        dict
+            The computed result as dictionary. This will be either returned
+            to the user or stored in the storage by calling the store method
+            with this as a parameter. The dictionary has the following keys:
+            - data : the correlation values between the two parcellations as
```

Comment:
* use '*' for items
* leave one blank line before the first list item
* leave one blank line after the last list item